### PR TITLE
added spongebob's quote

### PR DESCRIPTION
**Pull request by rizkyarlin**
Here I added my fav quotes from spongebob

### CODE REVIEW

File: data/data.json [hunk 1]
```diff
@@ -3895,5 +3895,12 @@
         "type": "movie",
         "year": 2011
  
+    },
+    {
+        "quote": "Hidup seperti Larry!",
+        "movie": "Spongebob Squarepants",
+        "type": "movie",
+        "year": 2002
```
Comment (andiwijayaid) [inline data/data.json]: wrong year, please fix it!

**Review by andiwijayaid — CHANGES_REQUESTED** [review]
asap please

File: data/data.json (hunk 1 above)
Comment (rizkyarlin) [reply data/data.json]: fixed!

File: data/data.json [hunk 2]
```diff
@@ -3900,7 +3900,7 @@
         "quote": "Hidup seperti Larry!",
         "movie": "Spongebob Squarepants",
         "type": "movie",
-        "year": 2002
+        "year": 2004
```
Comment (andiwijayaid) [inline data/data.json:3903]: good job

**Review by andiwijayaid — APPROVED** [review]

**Pull request merged**